admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/createtenantwizard/template.vue

```diff
@@ -99,6 +99,27 @@
 </template>
 
 <script>
+    const nameAvailableDebouncer = (function() {
+        let timeout
+        let oldReject
+        return {
+            call: (func, wait) => new Promise((resolve, reject) => {
+                let context = this, args = arguments
+                let later = function() {
+                    resolve(func.apply(context, args))
+                }
+
+                if (timeout) {
+                    clearTimeout(timeout)
+                    oldReject()
+                }
+
+                oldReject = reject
+                timeout = setTimeout(later, wait)
+            })
+        }
+    })()
+
```

Comment:
honestly this whole thing seems out of place. It looks like it's a generalised debouncer instead of a specific one.

So I'd rename it and move it into utils.js (or similar?)

Reply:
Nice suggestion, done.